main/broadcasters.c

```diff
@@ -548,6 +561,25 @@ static void mijia_sensor_metadata_get(uint8_t *adv_data, size_t adv_data_len,
                 le16toh(*(uint16_t *)(mijia_data_entry->data + 2)) / 10.0);
             cb("Humidity", s, ctx);
         }
+        else if (mijia_data_entry->data_type == MIJIA_SENSOR_DATA_TYPE_MOTION_LIGHT)
+        {
+            uint32_t val = mijia_data_entry->data[0]
+                | (mijia_data_entry->data[1] << 8) | (mijia_data_entry->data[2] << 16);
+            if (device_type == MIJIA_DEVICE_TYPE_CGPR1)
+            {
+                sprintf(s, "%u", val);
+                cb("Illuminance", s, ctx);
+            }
+            else if (device_type == MIJIA_DEVICE_TYPE_MJYD02YL)
+            {
+                cb("Light", val == 100 ? "1" : "0", ctx);
+            }
+            else if (device_type == MIJIA_DEVICE_TYPE_RTCGQ02LM)
+            {
+                cb("Light", val == 256 ? "1" : "0", ctx);
+            }
```

Comment:
Coding style nitpicking: Please remove the curly brackets from single line blocks (unless the statement above it is multi line).

Reply:
Please just fix this and I'll merge

Reply:
done